assets/src/edit-story/components/header/title.js

```diff
@@ -25,26 +27,42 @@ const Input = styled.input`
 
 function Title() {
 	const {
-		state: { story: { title, status } },
+		state: { story: { title, slug } },
 		actions: { updateStory },
 	} = useStory();
 
+	const { storyId } = useConfig();
+
 	const handleChange = useCallback(
 		( evt ) => updateStory( { properties: { title: evt.target.value } } ),
 		[ updateStory ],
 	);
 
+	const handleBlur = useCallback(
+		() => {
+			if ( ! slug || slug === storyId ) {
+				const clearSlug = encodeURIComponent( cleanForSlug( titleFormatted( title ) ) ) || storyId;
```

Comment:
I don't understand the name `clearSlug` here.

Reply:
Changed in https://github.com/ampproject/amp-wp/pull/4122/commits/27a750e4e53b9418c7ea0f6a7adbfdd1806c60d7